src/autocomplete/index.ts

```diff
@@ -22,8 +32,8 @@ class Autocomplete {
     this.flux.on('sayt:select_active', this.selectActive);
   }
 
-  activationTargets() {
-    return this.root.querySelectorAll('.gb-autocomplete-target');
+  activationTargets(): NodeListOf<HTMLElement> {
+return <any>this.root.querySelectorAll('.gb-autocomplete-target');
```

Comment:
can this be one line?

Reply:
I've been trying to keep away from doing arrow functions in the components unless we need to in order to preserve `this` bindings

Reply:
tests are dependant on groupby/flux-capacitor#60